colossalai/pipeline/schedule/one_f_one_b.py

```diff
@@ -32,20 +32,23 @@ def __init__(
         num_microbatches: Optional[int] = None,
         microbatch_size: Optional[int] = None,
         enable_metadata_cache: bool = True,
+        overlap_p2p=None,
     ) -> None:
         """1F1B pipeline schedule.
 
         Args:
             stage_manager (PipelineStageManager): Pipeline stage manager
             num_microbatches (Optional[int], optional): The number of microbatches. If not provided, it will be derived from microbatch size. Defaults to None.
             microbatch_size (Optional[int], optional): Microbatch size. If num_microbatches is provided, this will be ignored. Defaults to None.
+            overlap_p2p: Placeholder for interface consistency
         """
         super().__init__(stage_manager)
         assert (
             num_microbatches is not None or microbatch_size is not None
         ), "Either num_microbatches or microbatch_size should be provided"
 
-        self.comm = PipelineP2PCommunication(stage_manager)
+        self.comm = PipelineP2PCommunication(stage_manager, overlap_p2p=False)
```

Comment:
As this is always false, we'd better don't accept this arg (line 35), cause it's a little confusing.